runtime/altair/src/lib.rs

```diff
@@ -490,6 +490,7 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 					RuntimeCall::Loans(pallet_loans::Call::propose_write_off_policy{..}) |
 					RuntimeCall::Loans(pallet_loans::Call::apply_write_off_policy{..}) |
 					RuntimeCall::Loans(pallet_loans::Call::update_portfolio_valuation{..}) |
+                    RuntimeCall::Loans(pallet_loans::Call::apply_transfer_debt { .. }) |
```

Comment:
Missing propose here

Reply:
I was not sure about how to interpret that. Could `propose()` imply a transfer if later somebody apply it? Is `transfer_debt()` taken into account as something that transfers?

Reply:
By definricion transfer debt does juts transfer internally, right? So it think it is fine. 

If you think differently, then I would remove also the apply one.

Reply:
If transferring the debt locally counts as a transfer, then we should not add it
If it's not a transfer, we should add it

Reply:
He is off. Let's go with adding it!

transferring is just concerned for the acting account to move their funds which will not be the case here.

Reply:
Make sense!

Reply:
Agreed with @mustermeiszer, transferring debt is separate (and much less impactful from security perspective) from transferring stablecoins.